tool_commands/command_box.py

```diff
@@ -49,6 +49,9 @@ def __init__(self, iface, ts_datasource):
 
         self.ts_datasource = ts_datasource
 
+        # initialize plugin directory
+        self.plugin_dir = plugin_dir
```

Comment:
Even kijken of 'ie nodig is. Volgens mij werd het jaren geleden in `self.icon_path` gebruikt, maar nu niet meer.